src/main/java/utils/GeoUtil.kt

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import domain.Location
+
+/**
+ * Measures the distance between to geographic coordinates in meters.
+ *
+ * @param l1 Location 1
+ * @param l2 Location 2
+ * @return Distance between two locations in meters.
+ */
+fun measureGeoDistance(l1: Location, l2: Location): Double {
```

Comment:
Add this as a function to location instead of a different function

Reply:
Fixed